cowprotocol/cowswap
# Setup for amplitude tracking

# Summary

Implements a new type of analytics https://analytics.amplitude.com/
Some of the events in this PR are
- Different page views
- Open currency select modal
- Pick a currency
- Import a new token

# Testing
- Go to the https://analytics.amplitude.com/cowswap-test/activity
- Open the Demo and create few events mentioned above or just change few pages
- In a few moments there should be data related to those event in the Amplitude console
![Screenshot from 2022-09-20 16-36-36](https://user-images.githubusercontent.com/34926005/191287264-0e3cf901-02ef-4ea7-8b8e-f42bf5d5c290.png)
- You can click on user id and see a View with events in chronological order of that user with more details
![Screenshot from 2022-09-20 16-37-39](https://user-images.githubusercontent.com/34926005/191287707-415d1d83-7157-4580-a4ad-8c65176cac3b.png)
